plugins/modules/nsxt_transport_node_collections.py

```diff
@@ -192,7 +191,27 @@ def check_for_update(module, manager_url, mgr_username, mgr_password, validate_c
     if existing_tnc is None:
         return False
     if existing_tnc['compute_collection_id'] == transport_node_collection_with_ids['compute_collection_id'] and \
-        existing_tnc['transport_node_profile_id'] != transport_node_collection_with_ids['transport_node_profile_id']:
+            existing_tnc['transport_node_profile_id'] != \
+            transport_node_collection_with_ids['transport_node_profile_id']:
+        return True
+    if existing_tnc.__contains__('description') and not transport_node_collection_with_ids.__contains__('description'):
```

Comment:
Bug Fixed - 2645684 - Updating 'Description' and 'Transport Node Profile' fields in "nsxt_transport_node_collections" module fails